### ISSUE TITLE
Lock search field

### ISSUE THREAD

**Issue by mkaatman** [monovertex/ygorganizer#29]
I thought it might be useful to have a small lock icon to the left of the magnifier icon on the search field. If it was clicked, you could switch between browse collection/browse cards and maintain the search term.

**Comment by monovertex** [comment]
Indeed, this would be useful. We can extend this to the future filtering as well.

I feel, however, that a lock is not the most intuitive way to approach this. When I first saw the word lock, it did not suggest to me the actual idea you described.

How often would this be used? Would it be a good idea to make it a default behavior?

On the other hand, we could have a global button that would be "apply the last filtering I made" that wouls also work if you closed the tab or logged out.

**Comment by mkaatman** [comment]
I know I've been typing in a term, realizing I'm in my collection and then clicking on browse cards and retyping the term quite a bit. 

Maybe we don't need a lock, maybe making it the default is sufficient and they simply use the 'x' to clear the term if they want to.

**Comment by monovertex** [comment]
Yeah, that's what I was thinking. Okay, default it us then. I think we should also remember these between session, via cookie or something.

Many times I wanted to resume my search and had to type it all out again.
